web/static/js/components/css_modules/user_list.css

```diff
@@ -12,3 +12,17 @@
   white-space: nowrap;
   overflow-x: scroll;
 }
+
+:global(.basic.segment).index {
+  :global(.wrap) {
+    display: inline-grid;
```

Comment:
hmmm... if we go this route, this will make the vertical columns fill up first. i think we can get away with a the rows wrapping wrapping by setting a max-height and overflow-y: scroll on the list element, and setting the list items to `display: inline-block`. That way, a new row will only be introduced once adding a new user caused a line break